message_class.py

```diff
@@ -0,0 +1,77 @@
+class Message:
+    """
+    Message class. Represents a virtual message
+
+    :ivar id: id to the message.
+    :ivar sender: The sender's name.
+    :ivar body: The message content.
+
+    """
+    def __init__(self, message_id: int, message_sender: str, message_body: str):
```

Comment:
Your issue with read_inbox (deleting all read messages) can maybe be resolved here.